holoviews/operation/datashader.py

```diff
@@ -123,6 +73,77 @@ class aggregate(Operation):
                                         doc="""
         The type of the returned Elements, must be a 2D Dataset type.""")
 
+    link_inputs = param.Boolean(default=True, doc="""
+         By default, the link_inputs parameter is set to True so that
+         when applying shade, backends that support linked streams
+         update RangeXY streams on the inputs of the shade operation.""")
+
```

Comment:
Maybe expand to say why one might want to set it to False?  E.g. if you reuse objects in different cells of a notebook and find them inappropriately linked?